packages/atlas/src/hooks/useNftState.ts

```diff
@@ -1,19 +1,50 @@
 import BN from 'bn.js'
 
+import { useBids } from '@/api/hooks/bids'
 import { BasicNftFieldsFragment } from '@/api/queries'
 import { useBlockTimeEstimation } from '@/hooks/useBlockTimeEstimation'
 import { NftSaleType } from '@/joystream-lib'
 import { useJoystreamStore } from '@/providers/joystream'
 import { useUser } from '@/providers/user'
+import { SentryLogger } from '@/utils/logs'
 
 export type EnglishTimerState = 'expired' | 'running' | 'upcoming' | null
 
 export const useNftState = (nft?: BasicNftFieldsFragment | null) => {
-  const { activeMembership } = useUser()
+  const { activeMembership, memberId } = useUser()
   const { currentBlock, currentBlockMsTimestamp } = useJoystreamStore()
   const { convertBlockToMsTimestamp } = useBlockTimeEstimation()
 
+  const { bids: userBids } = useBids(
+    {
+      where: {
+        isCanceled_eq: false,
+        nft: { id_eq: nft?.id },
+        bidder: { id_eq: memberId },
+      },
+    },
+    {
+      fetchPolicy: 'cache-and-network',
+      skip: !nft?.id || !memberId,
+      onError: (error) =>
+        SentryLogger.error('Failed to fetch member bids', 'useNftState', error, {
+          data: {
+            nft: nft?.id,
+            member: memberId,
+          },
+        }),
+    }
+  )
+
   const auction = nft?.transactionalStatusAuction || null
+  const auctionId = auction && auction.id
+  const unwithdrawnUserBids = userBids?.filter(
+    (bid) =>
+      bid.auction.auctionType.__typename === 'AuctionTypeOpen' &&
+      (auction || bid.auction.id !== auctionId) &&
+      bid.auction.winningMemberId !== memberId
+  )
+  const bidFromPreviousAuction = unwithdrawnUserBids?.[0]
```

Comment:
Why is this happening in this hook now? Doesn't seem like data about unwithdrawn bids from previous auctions is needed for every nft/video tile, we still want to keep that in NftWidget only

Reply:
What about withdrawing past bids from context menu?

Reply:
I'm pretty sure so far we only allowed withdrawing bids from context menu if the bid was made in the current auction. Is that not correct?